src/autoloads/EnemyUtil.gd

```diff
@@ -0,0 +1,3 @@
+extends Node
+
+var aggro_array : Array[CharacterBody2D]
```

Comment:
while a lot of enemies will be CharacterBody2Ds, lots could also not be, for e.g. turrets. i don't see why we'd need to restrict this to just CharBodies.

Reply:
Whoops